crates/codegen/parser_templates/src/rust/kinds.rs

```diff
@@ -0,0 +1,20 @@
+use serde::Serialize;
```

Comment:
IIUC, these files are completely generated in `CodeGenerator::write_rust_lib_sources` .. Is there another use case?

+ others.

Reply:
Yes - to allow the template project to be 'live'